### PR TITLE
pep8 related fixes

### PR DESCRIPTION
**Pull request by parthivgls**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You can remove classes initialization.

File: medical/models/medical_appointment.py [hunk 1]
```diff
@@ -97,39 +101,44 @@ class MedicalAppointment(orm.Model):
...
+            vals['appointment_hour'] + ':' + \
+            vals['appointment_minute']
+        vals['appointment_date'] = datetime.strptime(date_time_str,
+                                                     '%Y-%m-%d %H:%M')
 
         val_history['name'] = uid
         val_history['date'] = time.strftime('%Y-%m-%d %H:%M:%S')
-        val_history['action'] = "--------------------------------  Changed to Comfirm  ------------------------------------\n"
-
+        val_history['action'] = "--------------------------------"
```
Comment (yvaucher) [inline medical/models/medical_appointment.py:112]: See my refactoring here 
https://github.com/OCA/vertical-medical/pull/6/files#diff-3342df827adae2a46375586cb86b0046R29

```
HISTORY_ACTION_FORMAT = "-" * 32 + "  %s  " + "-" * 32 + "\n"
```

Then:

```
 val_history = {
        'name': uid,
        'date': time.strftime('%Y-%m-%d %H:%M:%S'),
        'action': HISTORY_ACTION_FORMAT % "Changed to Confirm"
    }
```

**Comment by nhomar** [comment]
@pedrobaeza I will merge this and inmediatly I will make a new PR for that. (which is Odoo standard not PEP8) but you are right.

**Comment by nhomar** [comment]
@parthivgls can you make a new PR with only what Pedro is mentioning please.

**Pull request merged**